packages/query/codegen.ts

```diff
@@ -13,7 +13,8 @@ const config: CodegenConfig = {
 	schema: process.env.GRAPHQL_SCHEMA_FILE ?? [
 		{
 			[ generateGraphqlUrl(
-				process.env.NEXT_PUBLIC_WP_HOME_URL,
+				process.env.NEXT_PUBLIC_WP_SITE_URL ||
+					process.env.NEXT_PUBLIC_WP_HOME_URL,
 				process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT
 			) ]: {
```

Comment:
This whole things is getting too messy, and not _just_ because I hate the `generate()` function.